src/ReleaseHistory.md

```diff
@@ -2,6 +2,7 @@
 
 ## **v3.1.0** (UNRELEASED)
 
+* BREAKING: Properties changed from `int`(32bit) to `BigInteger`(unlimited): `Address.AbsoluteAddress`, `Address.RelativeAddress`, `Address.Length`, `Address.OffsetFromParent`, `Artifact.Offset`, `Artifact.Length`, `Invocation.ExitCode`, `Invocation.ExitSignalNumber`, `LocationRelationship.Target`, `Region.ByteOffset`, `Region.ByteLength`, `Result.OccurrenceCount`. [#2572](https://github.com/microsoft/sarif-sdk/pull/2572)
```

Comment:
>`Invocation.ExitCode`, `Invocation.ExitSignalNumber` [](http://example.com/codeflow?start=215&length=52)

Let me know if we want to use int32 instead for:
`Invocation.ExitCode`, `Invocation.ExitSignalNumber`
I don't think there is clear indication of limitation in SARIF spec,
e.g. if user has a method that has return value defined as C# `BigInteger`  and he use that return value as `ExitSignalNumber`.
So I currently use `BigInteger`

Reply:
Excellent points. I have the exact same opinion.